task/dev9/src/com/github/zubarevladimir/IniMapInitializer.java

```diff
@@ -0,0 +1,51 @@
+package com.github.zubarevladimir;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Contains method for initialize map according to string.
+ */
+public class IniMapInitializer {
+
+  /**
+   * Initialize map according to string made of .ini file.
+   *
+   * @param text string made of .ini file.
+   * @return Map - contains key(section) and value(Map - key(key) and value(value)) according to
+   * given string.
+   */
```

Comment:
no need in class which has only one public method and has no fileds, has to be joined with class Searcher